models/tasks.js

```diff
@@ -126,7 +126,7 @@ const fetchSelfTask = async (taskId, userId) => {
  * @return {Promise<tasks|Array>}
  */
 
-const fetchUserTasks = async (username, statuses = []) => {
+const fetchUserTasks = async (username, statuses = [], field, order) => {
```

Comment:
**Suggestion:** we can place statues as the last argument so that it can be skipped at line 190

Reply:
majority of the time we would be using the status parameter and field, order less frequently. So let's have status on the 2nd.